AppCenter/AppCenterTests/MSLogDBStorageTests.m

```diff
@@ -622,36 +625,168 @@ - (void)testAddLogsDoesNotExceedCapacity {
   }
 }
 
-- (void)testOldestLogsAreDeletedFirstWhenCapacityIsReached {
+- (void)testOldestNormalPriorityLogsAreDeletedFirstWhenCapacityIsReachedAndNewNormalPriorityLogComes {
```

Comment:
I would suggest using shorter such as `putNormalPriorityLogPurgesOldestNormalPriorityLogs`.

Reply:
Everyone has different preferences. :cry:

Reply:
Let's not block on preferences.